src/main/java/com/truelayer/java/paymentsproviders/IPaymentsProvidersApi.java

```diff
@@ -14,5 +15,6 @@ public interface IPaymentsProvidersApi {
 
     @GET("/payments-providers/{id}")
     CompletableFuture<ApiResponse<PaymentsProvider>> getProvider(
-            @Path("id") String providerId, @Query("client_id") String clientId);
```

Comment:
This is a breaking change that requires a major version bump. Please define a new version `12.0.0` [here](https://github.com/TrueLayer/truelayer-java/blob/main/gradle.properties#L4)